build.sbt

```diff
@@ -13,7 +13,7 @@ val collectionsScalaVersionSettings = Seq(
 val commonSettings = Seq(
   organization := "ch.epfl.scala",
   version := "0.10.0-SNAPSHOT",
-  scalaVersion := "2.12.4",
+  scalaVersion := "2.13.0-M2",
```

Comment:
Without that change I'm getting:

```
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 	::          UNRESOLVED DEPENDENCIES         ::
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 	:: ch.epfl.scala#collection-strawman_2.12;0.10.0-SNAPSHOT: not found
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 
[warn] 	Note: Unresolved dependencies path:
[warn] 		ch.epfl.scala:collection-strawman_2.12:0.10.0-SNAPSHOT
[warn] 		  +- ch.epfl.scala:timebenchmark_2.12:0.10.0-SNAPSHOT
```
Am I doing something wrong or was it just simply overlooked?